news/1012.bugfix

```diff
@@ -0,0 +1,2 @@
+- Fix tests in Travis.
```

Comment:
We usually do not include "internal" changes like the CI setup in the changelog. Not sure if Towncrier has an "internal" category.

Reply:
I agree that changes in CI should not be in Changes. I put it only because of the Changelog verifier. It seems that the twisted doesn't have an "internal" category: 

https://github.com/twisted/towncrier#news-fragments

Can I remove it even if the Changelog verifier turns red?